app/launcher/wallet.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+
+#while true; do echo "I'm a wallet"; sleep 1; done
+echo "I'm a wallet"
+sleep 20
```

Comment:
Idea was to (at least have) temporary manual tests using files like this 🤔

Reply:
Oui :+1: